Projects/AccountSDKIOSWeb/Tests/AccountSDKIOSWebTests/URLBuilderTests.swift

```diff
@@ -66,7 +66,7 @@ final class URLBuilderTests: XCTestCase {
     
     func testLoginURLWithMFAIncludesACRValues() {
         let sut = URLBuilder(configuration: Fixtures.clientConfig)
-        let authRequest = URLBuilder.AuthorizationRequest(loginHint: "", extraScopeValues: [])
+        let authRequest = URLBuilder.AuthorizationRequest(loginHint: "", assertion: "", extraScopeValues: [])
         let loginURL = sut.loginURL(authRequest: authRequest, authState: AuthState(mfa: .otp))
         
         XCTAssertEqual(loginURL?.scheme, "https")
```

Comment:
is it valuable to add test with assertion?

Reply:
I added a unit test for assertion string 🚀